inference-engine/tests/functional/plugin/shared/src/low_precision_transformations/reshape_transformation.cpp

```diff
@@ -52,6 +50,11 @@ void ReshapeTransformation::SetUp() {
 
 TEST_P(ReshapeTransformation, CompareWithRefImpl) {
     Run();
+
+    const auto params = std::get<3>(GetParam());
+    const auto actualPrecision = getRuntimePrecisionByType(params.layerType);
+    const auto expectedPrecision = params.expectedKernelType;
+    EXPECT_EQ(actualPrecision, expectedPrecision);
```

Comment:
Could you please override `Run()` method instead of writing additional code inside `TEST_P`?

Reply:
done, thanks